examples/dtls-echo/Makefile

```diff
@@ -49,6 +49,8 @@ CFLAGS += -DDEVELHELP
 # NOTE: Add the package for TinyDTLS
 USEPKG += tinydtls
 
+USEMODULE += tinydtls
```

Comment:
(this is not properly cleaned up in master, so we can't use it as an example ;-))